src/Sylius/Bundle/AdminBundle/Resources/config/routing/taxon.yml

```diff
@@ -0,0 +1,38 @@
+sylius_admin_taxon:
+    resource: |
+        alias: sylius.taxon
+        section: admin
+        templates: SyliusAdminBundle:Crud
+        only: ['update', 'delete']
+        redirect: update
+        vars:
+            all:
+                templates:
+                    form: SyliusAdminBundle:Taxon:_form.html.twig
+                    subheader: sylius.ui.manage_taxons
+                    icon: folder
+    type: sylius.resource
+
+sylius_admin_taxon_index:
+    path: /taxons/
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.taxon:indexAction
+        _sylius:
+            template: SyliusAdminBundle:Taxon:index.html.twig
+            section: admin
+            repository:
+                method: findRootNodes
+    host: "localhost"
```

Comment:
I understand the reasoning behind that, but I'm pretty sure that with this configuration when my application will be located under other host than `localhost` e.g. `mydomain.com` it will not work. Have you verified that?